app/controllers/api_controller/rates.rb

```diff
@@ -8,7 +8,7 @@ def rates_query_resource(object)
     end
 
     def create_resource_rates(_type, _id, data = {})
-      ChargebackRateDetail.create(data)
+      ChargebackRateDetail.create!(data)
```

Comment:
@abellotti do you think this is ok or would you rather raise an BadRequestError with the errors of the AR object?
Doing it this way the response would be:
```
{"error":{"kind":"internal_server_error","message":"Validation failed: Group can't be blank, Source can't be blank","klass":"ActiveRecord::RecordInvalid"}}                                                                                                                              
````